modules/mobile/config/routes.rb

```diff
@@ -8,6 +8,7 @@
     get '/appointment_requests/:appointment_request_id/messages', to: 'appointment_request_messages#index'
     get '/appointments', to: 'appointments#index'
     put '/appointments/cancel/:id', to: 'appointments#cancel'
+    post '/appointments/requests', to: 'appointments#create'
```

Comment:
This endpoint is specifically for create an appointment request. Do you think `create` is too generic in that caase?

Reply:
I'm just thinking ahead to direct appointment scheduling. If we intend to have different routes for creating appointments and appointment requests, then I'd suggest naming this action `create_request` or something. Perhaps @aherzberg has an opinion since he's been thinking about this project more than I have.

Reply:
I'm good with creating the requests in the appointments controllers since we already mix the two in the index action out of necessity. It just means when we do add direct scheduling that the create action will need a `type` parameter added to specify which. That will also be consistent with the other endpoints being added for appointment requests, like the facility eligibility endpoint will have a `type` parameter to specify direct or request.

Reply:
If that's the case, should we change the route to just `post '/appointment'` or something like that to avoid anchoring to the word "request"?

Reply:
Yes, I think that would be good.

Reply:
Works for me